src/categorical_algebra/DPO.jl

```diff
@@ -12,15 +12,19 @@ Apply a rewrite rule (given as a span, L<-I->R) to a ACSet
 using a match morphism `m` which indicates where to apply
 the rewrite.
 """
-function rewrite_match(L::ACSetTransformation, R::ACSetTransformation,
-                       m::ACSetTransformation)::ACSet
+function rewrite_match_maps(L::ACSetTransformation, R::ACSetTransformation,
+                         m::ACSetTransformation)::Vector{ACSetTransformation}
   dom(L) == dom(R) || error("Rewriting where L, R do not share domain")
   codom(L) == dom(m) || error("Rewriting where L does not compose with m")
-  (k, _) = pushout_complement(L, m)
-  l1, _ = pushout(R, k)
-  return codom(l1)
+  (ik, kg) = pushout_complement(L, m)
+  rh, kh = pushout(R, ik)
+  return [ik, kg, rh, kh]
```

Comment:
Should return tuple rather than vector. If the functions can be given meaningful names (not sure), it would be nice to return a named tuple.